source/core/output_buffers/ut_output_table_buffer.tpb

```diff
@@ -99,13 +100,26 @@ create or replace type body ut_output_table_buffer is
       commit;
     end;
 
-  begin
-    loop
-      select a.message_id, ut_output_data_row(a.text, a.item_type)
+    procedure remove_buffer_info is
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Avoid procedures without parameters. ([plsql:ProcedureWithoutParametersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AProcedureWithoutParametersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=816&issues=AWhCR0RKSi5GWQS9iTtY&open=AWhCR0RKSi5GWQS9iTtY)

[ISSUE_KEY]: AWhCR0RKSi5GWQS9iTtY